ci/tlk.ps1

```diff
@@ -823,6 +823,10 @@ class TlkRecipe
             $DpkgBuildPackageArgs += @('-a', $this.Target.DebianArchitecture())
         }
 
+        # Strip first so the included binary is identical for deb and rpm
+        & 'strip' $Executable | Out-Host
+
+        $env:DEB_BUILD_OPTIONS = "nostrip"
```

Comment:
nitpick: Don’t forget a comment explaining why we disable strip at this place. "Binary already stripped by Cargo in production profile"